src/views/AuthPage.vue

```diff
@@ -0,0 +1,26 @@
+<script>
+import { defineComponent } from 'vue';
+
+import RegisterForm from '@/components/Auth/RegisterForm.vue';
+import LoginForm from '@/components/Auth/LoginForm.vue';
+
+export default defineComponent({
+  name: 'AuthPage',
+  components: { RegisterForm, LoginForm },
+  computed: {
+    currentRouteName() {
+      return this.$route.name;
+    }
+  }
+});
+</script>
+
+<template>
+  <template v-if="currentRouteName === 'Register'">
+    <RegisterForm />
+  </template>
+
+  <template v-if="currentRouteName === 'Login'">
+    <LoginForm />
+  </template>
```

Comment:
How about we do a nested route here instead and let our router handle the specifics? 😎 

```suggestion
  <router-view />
```

Also if there are no common elements between these, you could just as well have two separate pages for these and that's also fine.

Reply:
In fact, I've had it done the way You say initially..
Then I thought that it would be smart to lazy load only one component `AuthPage`  through the router and let it handle the rest.
Seems like sometimes it's just better to go with our initial gut feeling :smiley:

Reply:
It's nice to see you are experimenting though!
The router will lazy load them anyway and it's much more clear what's going on this way 😄